.buildkite/bench_pipeline.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+steps:
+  - label: "bench-x86_64"
+    commands:
+      - pytest rust-vmm-ci/integration_tests/test_benchmark.py -s
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+        image: "rustvmm/dev:v6"
+        always-pull: true
```

Comment:
These lines have a whitespace problem; they need to be indented.

This is what triggers the following error in the Buildkite pipeline:
```
2020-07-24 11:44:30 WARN   POST https://agent.buildkite.com/v3/jobs/033d13f0-9ad6-4863-b9b1-e6411cb53262/pipelines: 422 `image` is invalid: configuration must be defined with a hash (Attempt 1/60 Retrying in 5s)
```